content/Watsonx/WatsonxData/Level-4/204.md

```diff
@@ -215,36 +215,36 @@ exit
 ```
 8. Go back to the IBM watsonx.data UI. From the **Data manager** section, Expand the 
 **iceberg_data** catalog in the **Catalogs associated** area and then expand the **ingest** schema.
-    ![](./images/205/ingest.png)
+    ![](../images/204/ingest.png)
 
 9. Select the **yellow_tripdata_2021_01_localfile** table to see the table definition. Select the 
 **Table schema** tab to view the table schema if not already selected.
-    ![](./images/205/table-schema.png)
+    ![](../images/204/table-schema.png)
 
 10. Select the **Data sample** tab to see a sample of the data.
-    ![](./images/205/data-sample.png)
+    ![](../images/204/data-sample.png)
 
 11. Now you can use the UI to run a query against this imported data. Select the 
 **Query workspace (SQL)** icon from the left side menu.
-    ![](./images/205/sql-workspace.png)
+    ![](../images/204/sql-workspace.png)
 
 12. On the line where the **yellow_tripdata_2022_01_localfile** table is located, click the 
 **Query templates** icon at the end of the name. This will display a drop-down list. Select
 **Generate SELECT**
-    ![](./images/205/generate-select.png)
+    ![](../images/204/generate-select.png)
 
 This generates SQL SELECT statement in the window to the right of the table name.
-    ![](./images/205/select-statement.png)
+    ![](../images/204/select-statement.png)
 
 13.In order to review the results, execute the query by clicking on the blue 
 **Run on presto-01 button**.
-    ![](./images/205/run-on-presto.png)
+    ![](../images/204/run-on-presto.png)
 
 ### Summary
 You have now completed the lab!
 In this lab you ingested data by using the watsonx.data ibm-lh command-line utility and ran 
 a SQL statement to query the data that was ingested.
 
-### Congratulations, you've reached the end of lab 205.
+### Congratulations, you've reached the end of lab 204.
 
 Click, [lab 106](/watsonx/watsonxdata/106) to start next lab.
```

Comment:
We might need to remove this since it links out to the _Lab 106: for Federated Queries._ 

...can be a follow-up

```suggestion

```